Source/Data/Blackjack.Data/Repos/UserRepo.cs

```diff
@@ -77,14 +84,33 @@ public async Task<List<UserEntity>> GetUsersBySoundex(string soundex)
             }
         };
 
-        DataTable queryData = await base.Get(StoredProcedures.GetUser, parameters);
+        DataTable queryData = await base.Get(StoredProcedures.GetUserBySoundex, parameters);
 
         List<UserEntity> userEntities = CreateUserEntity(queryData);
 
         return userEntities;
     }
 
-    public async Task<List<UserEntity>> UpdateUser(UserEntity user)
+    public async Task<UserEntity> GetUserByID(int userID)
+    {
+        List<SqlParameter> parameters = new List<SqlParameter>()
+        {
+            new SqlParameter()
+            {
+                ParameterName = $"@{nameof(UserEntity.UserID)}",
+                SqlDbType = SqlDbType.VarChar,
```

Comment:
This is a Int in the Db. Did this code work for you successfully in localhost?

Reply:
It did work curiously enough. I updated it though